src/test/resources/db/test/coordinator_sign_plan_data.sql

```diff
@@ -0,0 +1,28 @@
+-- Insert into practitioner
```

Comment:
This is mostly a duplicate of the oasys_assessment_pk_data.sql - I wonder if we really need this, and suggest the test should just use the two existing oasys data and cleanup SQL files, and one that just sets the agreement_status of the plan_version to AGREED

Reply:
resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/117/commits/87a804e371495a39747b67c25c4893ed37298f0d